test/testcafe/spec/ChallengeAuthenticatorOktaVerify_spec.js

```diff
@@ -71,14 +71,17 @@ async function verifySelectAuthenticator(t, selectAuthenticatorPage, expectedRes
   await t.expect(JSON.parse(requestLog.body)).eql(expectedResponse);
 }
 
-async function setup(t, factorsCount = 3) {
+async function setup(t, factorsCount = 3, widgetOptions) {
```

Comment:
Maybe change to:
```js
async function setup( t, opts ) {
 const { factorsCount, ...widgetOptions } = opts;
 ...
 await t.expect(selectAuthenticatorPageObject.getFactorsCount()).eql(factorsCount ?? 3);
```

to avoid needing to pass 3 (which is the default value) on every `setup` call